ops/terraform/services/common/aurora-cluster.tf

```diff
@@ -117,7 +118,7 @@ resource "aws_db_parameter_group" "aurora_cluster" {
 
 resource "aws_rds_cluster_instance" "nodes" {
   count                           = local.rds_instance_count
-  auto_minor_version_upgrade      = false # minor cluster upgrades can cause downtime
+  auto_minor_version_upgrade      = true # minor cluster upgrades can cause downtime
```

Comment:
While this is _true_, it seems that this comment existed to justify keeping automatic minor version upgrades disabled. Maybe this comment ought to be removed, given that we are enabling this now and that [Aurora PostgreSQL 14.3.3 and above supports zero-downtime patching](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/USER_UpgradeDBInstance.PostgreSQL.html#USER_UpgradeDBInstance.PostgreSQL.Overview):
> [Zero-downtime patching] minimizes outages, and in most cases completely eliminates them.

Reply:
Sure, will do. Thanks.

Reply:
Removed.